### PR TITLE
Use Golang 1.15 for security fixes to Golang std library

### PR DESCRIPTION
**Pull request by umangachapagain**
Fixes BZ: 1906016

Signed-off-by: Umanga Chapagain <chapagainumanga@gmail.com>

### CODE REVIEW

**Review by jarrpa — APPROVED** [review]

File: functests/must-gather/must_gather_test.go [hunk 1]
```diff
@@ -3,6 +3,7 @@ package must_gather_test
 import (
 	"testing"
 
+	"github.com/onsi/ginkgo"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
```
Comment (jarrpa) [inline functests/must-gather/must_gather_test.go:8]: Do these need to be removed as well?

**Review by jarrpa — CHANGES_REQUESTED** [review]
Apart from failing tests (not sure if they're valid), I think there's a piece missing in the e2e tests.

File: functests/must-gather/must_gather_test.go (hunk 1 above)
Comment (umangachapagain) [reply functests/must-gather/must_gather_test.go]: Good catch. That's where the tests are failing.

**Comment by jarrpa** [comment]
Looks like there are still build failures, try running this locally to hunt it down.

**Review by jarrpa — APPROVED** [review]
/lgtm

**Pull request merged**